x

```diff
@@ -0,0 +1,85 @@
+Name:                 trino-worker-ccbc4647b-j58xd
```

Comment:
i don't see any secrets in this file, let's remove it

Reply:
🤦